docs/backends/googlecloudmonitoring.mdx

```diff
@@ -39,6 +39,14 @@ Odigos currently supports Standard (non-Autopilot) GKE clusters. Exporting into
   ✅ Logs
 </Accordion>
 
+- **GCP_TIMEOUT** `string` : Timeout. The request timeout in seconds for the Google Cloud Monitoring API.
```

Comment:
consider specifying the format here so it is explicit. For example, confused user can write `20` here and i guess it will fail to start with invalid config?

Reply:
updated the placeholder to be `10s`